docs/features/metrics.md

```diff
@@ -0,0 +1,54 @@
+# Metrics
+
+The metrics feature seeks to identify students who might be struggling in class by tracking metrics that signify possible risks. The feature is developed partially in response to the increasing number of classes done remotely in 2020, to help instructors find at risk students whom they should reach out to earlier in the course. Identifying at-risk students early would provide a better chance of getting them back on track in the course.
+
+
+## Usage Flow
+
+We envision the feature to be used by the instructors in this order. They would:
+
+1. Set up their course and assignments as per usual
+1. Set up [Risk Metrics](#risk-metrics) at the start of the course
+1. Be notified of new at risk students in their [Watchlist](#watchlist) on their course page
+1. Visit the [Watchlist](#watchlist), contact students if necessary, using it as a work list at the same time
+1. Refine the [Risk Metrics](#risk-metrics) as the course progresses
+
+## Risk Metrics
+
+![Metrics](/images/risk_metrics.png)
+
+From our interviews with instructors, we understand that different courses have different measures of whether a student is at risk. As such, a set of risk conditions together will define the course's risk metrics. We intend to add more conditions to the risk metrics in the future. Feel free to suggest them via our [GitHub Issues](https://github.com/autolab/Autolab/issues) page.
+
+### Risk Condition Rationale
+
+The conditions are designed to capture different characteristics of a possibly at risk student. In the sections that follow we attempt to explain the rationale behind each condition to aid with selecting the risk conditions.
+
+#### Students who have used *number* grace days by *date*.
+
+For courses that provides grace days, students who use many grace days early in the course tend to have issues managing the workload and/or their time. A good rule of thumb is that a student should not have used _**all**_ their grace days before the middle of the course.
+
+#### Students whose grades have dropped by *number* percent within *number* consecutive assignments within a category
+
+Identify students who have been slipping in their grades. Below are the underlying properties
+
+- windowed based on the *number* consecutive 
+- decrease must be consecutive
+- skips over no submissions
```

Comment:
nit: Maybe "no-submissions" (with the dash) instead of "no submissions"? I thought it meant "doesn't skip over any submissions" the first time I read it.